paymentsheet/src/main/java/com/stripe/android/link/LinkActivityViewModel.kt

```diff
@@ -223,49 +204,41 @@ internal class LinkActivityViewModel @Inject constructor(
         }
     }
 
-    fun linkScreenCreated() {
-        viewModelScope.launch {
-            val currentRoute = navController?.currentBackStackEntry?.destination?.route
-            if (currentRoute == null || currentRoute == LinkScreen.Loading.route) {
-                navigateToLinkScreen()
-            }
-        }
-    }
```

Comment:
This is not needed anymore:UI does not need to signal component rendering to trigger loading. Instead, we'll wait to render the component until we have the necessary info loaded.